src/components/home/aboutUsBlurb.tsx

```diff
@@ -0,0 +1,10 @@
+const AboutUsBlurb = () => {
+  return (
+    <div className="relative mt-[30vh] flex items-center justify-center">
+      <img src="home/aboutusframe.webp" width={1200} className="flex-shrink-0" />
+      <img src="home/aboutusimg.webp" width={580} className="absolute left-48" />
```

Comment:
```suggestion
      <img src="home/aboutusimg.webp" className="absolute left-48 w-[___vw]" />
```

use tailwind to control the img width